pkg/pull/interface.go

```diff
@@ -0,0 +1,21 @@
+package pull
+
+var p PullerInterface
```

Comment:
i like using the underscore to denote a global to prevent unintended conflicts with local vars

```suggestion
var _p PullerInterface
```